near-sdk/src/types/mod.rs

```diff
@@ -4,6 +4,9 @@ pub use self::vm_types::*;
 mod primitives;
 pub use self::primitives::*;
 
+mod account_id;
+pub use self::account_id::*;
```

Comment:
Style preference -- for public re-exports, I like to explicitlely list all things exported and avoid `*` -- that way, if the user learns the library by reading the source, they get a view of the public API at a glance. 

This is unrealated to the PR, and better be changed (if we do want to change this) in a follow-up.

Reply:
Yeah, I also usually prefer, forgot to cycle back